### PR TITLE
Create bare-bones pull request template

### PR DESCRIPTION
**Pull request by briandk**
This template needs fleshing out, but I figured a good place to start was making sure people run tests and making sure they update the Changelog.

Here's [some background][2] on [Issue Templates][2] and [Pull Request Templates][2].

[2]: https://help.github.com/articles/about-issue-and-pull-request-templates/

### CODE REVIEW

**Comment by glebec** [comment]
For organization's sake you may want to move this (and other GitHub-specific templates) into a `.github` folder – GH knows to check there.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- [ ] Run tests
+- [ ] Update the [`Changelog.md`][1] with a summary of your changes
```
Comment (JHilker) [inline PULL_REQUEST_TEMPLATE.md]: Nit: Might as well reference it as `CHANGELOG.md` to prevent any file case confusion?

**Comment by briandk** [comment]
@JHilker you may also want to think about issue templates

**Comment by CiGit** [comment]
Thanks, we should add that folder to https://github.com/prettier/prettier-vscode/blob/master/.vscodeignore

I'll do it

**Pull request merged**